pkgs/development/tools/misc/dart-sass/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, fetchFromGitHub
+, buildDartApplication
+}:
+
+buildDartApplication rec {
+  pname = "dart-sass";
+  version = "1.62.1";
+
+  src = fetchFromGitHub {
+    owner = "sass";
+    repo = pname;
+    rev = version;
+    hash = "sha256-U6enz8yJcc4Wf8m54eYIAnVg/jsGi247Wy8lp1r1wg4=";
+  };
+
+  pubspecLockFile = ./pubspec.lock;
```

Comment:
Was going to mention it was weird that dart-sass didn't include a `pubspec.lock`, but it looks like @lelgenio already opened a PR for that! So hopefully it'll be in the next release.

Reply:
Looks like upstream does not want to include it because dart-sass is also a library and must not impose specific version requirements, makes sense: https://github.com/sass/dart-sass/pull/1948#issuecomment-1539185332